aws-sns-topic/src/main/java/software/amazon/sns/topic/CreateHandler.java

```diff
@@ -54,9 +54,8 @@ protected ProgressEvent<ResourceModel, CallbackContext> handleRequest(
                                 throw new CfnGeneralServiceException(e);
                             }
                         })
-                        .stabilize(this::stabilizeSubscriptions)
```

Comment:
why are we removing `stabilize` step as well?

Reply:
We add this purely for contract test, to make the model read handler returns be the same as input in create. In our original implementation, we don't have this part of logic.